sycl/source/detail/platform_impl.cpp

```diff
@@ -120,104 +121,106 @@ vector_class<platform> platform_impl::get_platforms() {
   return Platforms;
 }
 
-struct DevDescT {
-  const char *devName = nullptr;
-  int devNameSize = 0;
-  const char *devDriverVer = nullptr;
-  int devDriverVerSize = 0;
+std::string getValue(/*const*/ std::string &AllowList, size_t &Pos,
```

Comment:
Why is `AllowList` not a const reference? There are now changes for it in the function.

```suggestion
std::string getValue(const std::string &AllowList, size_t &Pos,
```

Reply:
I intended to uncomment that before I checked it in.  <sigh>